src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -3581,10 +3601,11 @@ subroutine wrfv3_netcdf(fv3filenamegin)
                              add_saved,fv3filenamegin%dynvars,fv3filenamegin)
       call gsi_fv3ncdf_write(grd_fv3lam_tracer_ionouv,gsibundle_fv3lam_tracer_nouv, &
                              add_saved,fv3filenamegin%tracers,fv3filenamegin)
-      if( if_model_dbz ) then
+      if( if_model_dbz .or. if_model_fed ) then
```

Comment:
similarly, simpler just to test as " if( allocated(gsibundle_fv3lam_phyvar_nouv)"

Reply:
@TingLei-NOAA 
this is a bundle. I tried either allocated or associated, GSI compiling failed.